### ISSUE TITLE
non-discoverable passkey authenticatation not supported

### ISSUE THREAD

**Issue by oppressor1761** [bitwarden/mobile#3348]
### Production Build

- [X] I'm using the legacy Bitwarden app pubicly available in App Store / Play Store and I'm aware that Bitwarden Beta bugs should be reported in [bitwarden/android](https://github.com/bitwarden/android) or [bitwarden/ios](https://github.com/bitwarden/ios)

### Steps To Reproduce

1. on Windows/Mac, go to [webauthn.io](https://webauthn.io) and register a non-discoverable passkey using Browser extension
2. on Android, go to [webauthn.io](https://webauthn.io) and authenticate the passkey created in step 1, using the same username

### Expected Result

Authenticated successfully

### Actual Result

cannot authenticate the passkey created

### Screenshots or Videos

![baa074ccf6bcb04320e42861132912f](https://github.com/bitwarden/mobile/assets/163018825/1a562673-1fa3-4746-8c6a-016936df756f)
basically it says 

> no passkeys available on this device

### Additional Context

If it's a discoverable passkey registered in step 1, authentication would be successful in step 2.
If it's Proton Pass extension used in step 1, authentication would be successful in step 2 no matter the passkey is discoverable or non-discoverable.

### Operating System

Android

### Operating System Version

14

### Device

Google Pixel 8 Pro

### Build Version

2024.6.0(10746)

**Comment by sammbw** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by barbuslex** [comment]
I have the same issue on Oneplus 11 with Android 14 on Firefox and Chrome
